src/api/status/src/views/builds.hbs

```diff
@@ -51,15 +51,16 @@
               <div class="align-center">
                 <a id="build-sender" class="card-link" href="#">
                   <img
-                    class="avatar avatar-xs rounded-circle"
+                    class="avatar avatar-sm rounded-circle"
                     alt=""
                     src=""
                     id="build-sender-img"
                   />
                   <span id="build-sender-name"></span>
                 </a>
-                synchronized
+                merged
```

Comment:
I don't understand what the purpose of either `synchronized` or `merged` are here.  Can we get rid of this?  What is it telling us?  Every commit on GitHub was merged, or it wouldn't be there, so this seems redundant.